# Description: disable lyrics servers that are not working anymore, either
# because they are down or because they changed
# Upstream: https://gitlab.gnome.org/GNOME/rhythmbox/-/issues/1829
Index: rhythmbox/plugins/lyrics/LyricsSites.py
===================================================================
--- rhythmbox.orig/plugins/lyrics/LyricsSites.py
+++ rhythmbox/plugins/lyrics/LyricsSites.py
@@ -26,11 +26,11 @@
 
 
 #from AstrawebParser import AstrawebParser
-from WinampcnParser import WinampcnParser
+#from WinampcnParser import WinampcnParser
 from TerraParser import TerraParser
-from DarkLyricsParser import DarkLyricsParser
+#from DarkLyricsParser import DarkLyricsParser
 from JlyricParser import JlyricParser
-from JetlyricsParser import JetlyricsParser
+#from JetlyricsParser import JetlyricsParser
 
 from gi.repository import RB
 
@@ -39,10 +39,10 @@ gettext.install('rhythmbox', RB.locale_d
 
 lyrics_sites = [
 	#{ 'id': 'astraweb.com', 	'class': AstrawebParser, 	'name': _("Astraweb (www.astraweb.com)") 	},
-	{ 'id': 'winampcn.com', 	'class': WinampcnParser, 	'name': _("WinampCN (www.winampcn.com)") 	},
+	#{ 'id': 'winampcn.com', 	'class': WinampcnParser, 	'name': _("WinampCN (www.winampcn.com)") 	},
 	{ 'id': 'terra.com.br',		'class': TerraParser,		'name': _("TerraBrasil (terra.com.br)")		},
-	{ 'id': 'darklyrics.com',	'class': DarkLyricsParser,	'name': _("Dark Lyrics (darklyrics.com)")	},
+	#{ 'id': 'darklyrics.com',	'class': DarkLyricsParser,	'name': _("Dark Lyrics (darklyrics.com)")	},
 	{ 'id': 'j-lyric.net', 		'class': JlyricParser, 		'name': _("Jlyric (j-lyric.net)") 		},
-	{ 'id': 'jetlyrics.com', 	'class': JetlyricsParser, 	'name': _("Jetlyrics (jetlyrics.com)") 		}
+	#{ 'id': 'jetlyrics.com', 	'class': JetlyricsParser, 	'name': _("Jetlyrics (jetlyrics.com)") 		}
 ]
 
